alacritty/src/event.rs

```diff
@@ -472,12 +496,21 @@ impl<N: Notify + OnResize> Processor<N> {
                 );
             }
 
+            #[cfg(not(any(target_os = "macos", windows)))]
+            {
+                if event_loop.is_wayland()
+                    && !self.display.window.should_draw.load(Ordering::Relaxed)
+                {
+                    return;
+                }
+            }
```

Comment:
Maybe add a comment to this? Would probably be generous. Just something short and sweet to explain what we're doing here.

Suggestion:

```
// Skip rendering on Wayland if the next vblank hasn't been reached.
```

Alternatively if 'vblank' is too weird:

```
// Skip rendering on Wayland if the next frame hasn't been requested yet.
```

Maybe that's a little easier to understand, more accessible? I'll leave it up to you.

Reply:
It's about receiving frame callback from the server, though maybe requested is also fine.